website/blog/2021-07-21-Apache-APISIX-Kubernetes.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Apache APISIX x Kubernetes：恰到好处｜直播预告"
+keywords:
+- API 网关
+- APISIX
+- Kubernetes
+- 直播预告
+description: 直播预告：北京时间 2021 年 7 月 23 日星期五 00:00，Apache APISIX committer 张晋涛将在 Data on Kubernetes Community 的 Meetup 上，面向全球开发者做主题为《Run Apache APISIX in Kubernetes》的分享。
+tags: [technology]
+---
+<!--truncate-->
+
+北京时间 2021 年 7 月 23 日星期五 00:00， Apache APISIX committer 张晋涛将在 Data on Kubernetes Community 的 Meetup 上，面向全球开发者做主题为《Run Apache APISIX in Kubernetes》的分享。
+
+张晋涛将在本次演讲中分享在 Kubernetes 上运行 Apache APISIX，以及如何将 Apache APISIX 作为 Ingress Controller 来使用的经验。感兴趣的小伙伴不要错过，赶紧报名约起来~
+
+## 参与方式
+
+- 方式一：点击“阅读原文”跳转报名网页，注册报名。
+
+- 方式二：在直播开始时通过 Zoom 链接进入会议室聆听分享：[会议室直播间](https://zoom.us/webinar/tJYofuChrzktGtI3wr8SZHACRnNkxr5cWgny)。
+
+## 来不及看直播怎么办？
+
+Meetup 结束后，我们会在 Apache APISIX 的 bilibili 账号上传 Meetup 录播。欢迎大家关注Apache APISIX bilibili 账号 Apache-APISIX。
+
+## 讲师介绍
+
+张晋涛，支流科技云原生技术专家、Apache APISIX committer、Kubernetes ingress-nginx reviewer。他同时作为 containerd/Docker/Helm/Kubernetes/KIND 等众多开源项目 contributor 在开源社区活跃，是 『K8S 生态周报』的维护者之一。
+从业 7 年，张晋涛对 Docker 和 Kubernetes 等容器化技术有大量实践和深入源码的研究，是业内多个知名大会讲师，PyCon China 核心组织者之一。同时著有 《Kubernetes 上手实践》和 《Docker 核心知识必知必会》等专栏。运营有公众号：MoeLove。
+
+![讲师介绍-张晋涛](https://static.apiseven.com/202108/1630382172445-cf20986b-c939-497e-86a4-92da7064ae97.PNG)
+
+## 关于 Apache APISIX
+
+Apache APISIX 是一个动态、实时、高性能的开源 API 网关，提供负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。Apache APISIX 可以帮忙企业快速、安全的处理 API 和微服务流量，包括网关、Kubernetes Ingress 和服务网格等。
+
+全球已有数百家企业使用 Apache APISIX 处理关键业务流量，涵盖金融、互联网、制造、零售、运营商等等，比如美国航空航天局（NASA）、欧盟的数字工厂、中国航信、中国移动、腾讯、华为、微博、网易、贝壳找房、360、泰康、奈雪的茶等。
+
+200 余位贡献者，一同缔造了 Apache APISIX 这个世界上最活跃的开源网关项目。聪明的开发者们！快来加入这个活跃而多样化的社区，一起来给这个世界带来更多美好的东西吧！
+
+Apache APISIX 项目地址：https://github.com/apache/apisix
+Apache APISIX 官网：http://apisix.apache.org/zh/
```

Comment:
I think we do not need the website link, we've visited the site.

Reply:
fixed